<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Console</family>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRightBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="moveLeftBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shootButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Shoot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveUpBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="moveDownBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="newgameButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start new game</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPlayer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit game</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Scores</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player1Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="player3Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player4Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hitLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="text">
     <string>hitLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
